### PR TITLE
Submetrizable ==> Countable pseudocharacter.

### PR DESCRIPTION
**Pull request by Almanzoris**

### CODE REVIEW

File: theorems/T000520.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+uid: T000520
+if:
+  P000112: true
+then:
+  P000191: true
+---
+
+The topology contains a coarser topology which is {P53}, and {P53} implies {P191}.
```
Comment (StevenClontz) [inline theorems/T000520.md:9]:
```suggestion
For each point, consider the collection of balls of radius $1/n$ from the coarser {P53} topology.
```

Might make the argument a bit more direct (don't have to convince yourself that P191 holds for all finer topologies).

**Comment by StevenClontz** [comment]
~~This is correct, but can possibly be improved: is every submetrizable space a [$G_\delta$ space](https://topology.pi-base.org/properties/P000132)? Then we get points $G_\delta$ from https://topology.pi-base.org/theorems/T000502 (we already have submetrizable implies functionally hausdorff).~~

But wait, that's not possible: https://topology.pi-base.org/spaces?q=Submetrizable%2B%7E%24G_%5Cdelta%24+space

**Review by StevenClontz — APPROVED** [review]
Approved - consider merging my suggestion before merging the PR (or you can ignore my suggestion).

**Comment by Almanzoris** [comment]
> ~This is correct, but can possibly be improved: is every submetrizable space a [$G_\delta$ space](https://topology.pi-base.org/properties/P000132)? Then we get points G δ from https://topology.pi-base.org/theorems/T000502 (we already have submetrizable implies functionally hausdorff).~
> 
> But wait, that's not possible: https://topology.pi-base.org/spaces?q=Submetrizable%2B%7E%24G_%5Cdelta%24+space

Yeah, we just know that, at least, the closed sets of the coarser metrizable topology are $G_{\delta}$, like the points, as it's $T_1$.

**Comment by prabau** [comment]
Nice result.  We can now derive that 12 more spaces are not submetrizable.

**Comment by Almanzoris** [comment]
Thanks, sir.

**Pull request merged**